libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -327,7 +328,7 @@ def pull_cortx_image(self, worker_obj: LogicalNode):
 
     def deploy_cortx_cluster(self, sol_file_path: str, master_node_list: list,
                              worker_node_list: list, system_disk_dict: dict,
-                             git_tag) -> tuple:
+                             git_tag, namespace) -> tuple:
```

Comment:
can we read the default namespace here ? so that all tests need not to do modifications.

Reply:
done.